### PR TITLE
funny rc note about dm channels

### PR DESCRIPTION
**Pull request by Jupith**

### CODE REVIEW

File: pages/resources/user.mdx [hunk 1]
```diff
@@ -719,7 +719,7 @@ Returns a list of active [private channel](/resources/channel#channel-object) ob
   Create Private Channel
 </RouteHeader>
 
-One recipient creates or returns an existing [DM channel](/resources/channel#channel-object), no or multiple recipients create a [group DM channel](/resources/channel#channel-object). Returns a [private channel](/resources/channel#channel-object) object. Fires a [Channel Create](/topics/gateway-events#channel-create) Gateway event.
+One recipient creates or returns an existing [DM channel](/resources/channel#channel-object), no or multiple recipients create a [group DM channel](/resources/channel#channel-object). Returns a [private channel](/resources/channel#channel-object) object. Fires a [Channel Create](/topics/gateway-events#channel-create) Gateway event. (If multiple channels with a single recipient exist, the channel with the largest ID is returned)
```
Comment (dolfies) [inline pages/resources/user.mdx:722]:
```suggestion
One recipient creates or returns an existing [DM channel](/resources/channel#channel-object), no or multiple recipients create a [group DM channel](/resources/channel#channel-object). Returns a [private channel](/resources/channel#channel-object) object. Fires a [Channel Create](/topics/gateway-events#channel-create) Gateway event. 

If multiple channels with a single recipient exist, the most recent one is returned.
```

**Review by dolfies — CHANGES_REQUESTED** [review]

File: pages/resources/user.mdx (hunk 1 above)
Comment (Jupith) [reply pages/resources/user.mdx:722]: "Recent is meh here since were talking about literal miliseconds, largest ID is more clear cut

Comment (dolfies) [reply pages/resources/user.mdx:722]: Largest ID is unnecessarily verbose; recent is more in-line with how [snowflakes work](https://discord-userdoccers.vercel.app/reference#encryption).

Comment (Jupith) [reply pages/resources/user.mdx:722]: Docs should be as verbose as u can get :)

Comment (dolfies) [reply pages/resources/user.mdx:722]: Verbosity is appreciated when it adds additional context. Largest ID does not offer any extra information; it offers less information when taking the structure of a snowflake into account.

File: pages/resources/user.mdx [hunk 2]
```diff
@@ -721,6 +721,8 @@ Returns a list of active [private channel](/resources/channel#channel-object) ob
 
 One recipient creates or returns an existing [DM channel](/resources/channel#channel-object), no or multiple recipients create a [group DM channel](/resources/channel#channel-object). Returns a [private channel](/resources/channel#channel-object) object. Fires a [Channel Create](/topics/gateway-events#channel-create) Gateway event.
 
+If multiple channels with a single recipient exist, the most recent(Largest ID) channel is returned
```
Comment (dolfies) [inline pages/resources/user.mdx:724]:
```suggestion
If multiple channels with a single recipient exist, the most recent channel (largest ID) is returned
```

**Review by dolfies — APPROVED** [review]

**Pull request merged**